### DISCUSSION TITLE
App Variants

### DISCUSSION THREAD

**Discussion by Fabian-K** [parcel-bundler/parcel#8549] [Q&A]
Hi,

is there a way to build multiple variants of an application with parcel? Let´s say I have a module that takes care of user authentication and I have two different implementations of it, e.g. one using firebase auth and one using a different auth provider. It would be great to build two variants of the application, one using firebase and one using the other auth provider.

Aliases and targets seem to be interesting. However, it does not seem to be possible to define aliases only for targets.

Thanks, Fabian

**Comment by mischnic** [comment]
Currently no
